### PR TITLE
feat: add supportUs and some info in info.vue

### PR DESCRIPTION
**Pull request by treeleaves30760**

### CODE REVIEW

File: pages/supportUs.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-container class="px-6 px-md-0">
+    <v-row>
+      <v-col cols="12" md="7">
+        <v-img
+          src="https://eecampapi.nthuee.org/public/9fd9216f-ab48-48b7-85a6-66ab17fdc27a_NTHUEECamp.jpg"
```
Comment (ARui-TW) [inline pages/supportUs.vue:6]: Don't use this URL directly. Either use the URL saved in Vuex or use up load the picture to sites like Imgur and use that given URL.

File: pages/supportUs.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-container class="px-6 px-md-0">
+    <v-row>
+      <v-col cols="12" md="7">
+        <v-img
+          src="https://eecampapi.nthuee.org/public/9fd9216f-ab48-48b7-85a6-66ab17fdc27a_NTHUEECamp.jpg"
+        ></v-img>
+      </v-col>
+      <v-col cols="12" md="4">
+        <h1 class="text-center">支持我們</h1>
```
Comment (ARui-TW) [inline pages/supportUs.vue:10]: Don't use the heading tag to adjust text size. Use parameters defined in 
[`assets/css/typography.scss`](https://github.com/ColdeGarage/EECamp-web/blob/main/assets/css/typography.scss).

Comment (ARui-TW) [inline pages/supportUs.vue:10]: Also, don't use text-center. Please use flex.

File: pages/supportUs.vue (hunk 1 above)
Comment (treeleaves30760) [reply pages/supportUs.vue:6]: I decide to use imgur

File: pages/supportUs.vue [hunk 3]
```diff
@@ -1,30 +1,33 @@
 <template>
-  <v-container class="px-6 px-md-0">
+  <v-container class="">
```
Comment (ARui-TW) [inline pages/supportUs.vue:2]: Remove the empty class

File: pages/supportUs.vue [hunk 4]
```diff
@@ -1,30 +1,33 @@
...
-          <v-alert text border="bottom" color="warning">
-            <h3>粉絲專頁</h3>
-            <h5>
-              <a href="https://www.facebook.com/NTHUEECAMP/" class="muted">
-                https://www.facebook.com/NTHUEECAMP/
-              </a>
-            </h5>
-          </v-alert>
-        </div>
+        <v-row justify="space-around">
```
Comment (ARui-TW) [inline pages/supportUs.vue:12]: You don't need <v-row /> here, use `<div class="d-flex justify-space-around">` will be enough.

File: pages/supportUs.vue [hunk 5]
```diff
@@ -1,30 +1,33 @@
...
-        <v-img
-          src="https://eecampapi.nthuee.org/public/9fd9216f-ab48-48b7-85a6-66ab17fdc27a_NTHUEECamp.jpg"
-        ></v-img>
+      <v-col cols="12" md="8" class="d-flex align-center">
+        <v-img src="https://i.imgur.com/qF7zbDK.jpg"></v-img>
       </v-col>
       <v-col cols="12" md="4">
-        <h1 class="text-center">支持我們</h1>
+        <div class="text-Heading42">支持我們</div>
         <p class="font-weight-bold p-2">
```
Comment (ARui-TW) [inline pages/supportUs.vue:9]: There is no class `p-2`

File: pages/supportUs.vue [hunk 6]
```diff
@@ -1,30 +1,33 @@
...
-              <a href="https://www.facebook.com/NTHUEECAMP/" class="muted">
-                https://www.facebook.com/NTHUEECAMP/
-              </a>
-            </h5>
-          </v-alert>
-        </div>
+        <v-row justify="space-around">
+          <v-btn
+            x-large
+            large
```
Comment (ARui-TW) [inline pages/supportUs.vue:15]: Why use both `x-large` and `large`?

File: pages/supportUs.vue [hunk 7]
```diff
@@ -1,30 +1,33 @@
...
+            href="mailto:mailto:officialnthueecamp@gmail.com"
+            class="text-Heading24 ma-2"
+          >
+            粉專信箱
+          </v-btn>
+          <v-btn
+            x-large
+            large
```
Comment (ARui-TW) [inline pages/supportUs.vue:23]: Same as above.

**Review by ARui-TW — CHANGES_REQUESTED** [review]

**Comment by treeleaves30760** [comment]
I had changed the code

**Review by ARui-TW — APPROVED** [review]

**Pull request merged**